README.md

```diff
@@ -163,6 +163,13 @@ GitHub to deliver. For example, the default `Just the push event` works well
 with the email diff sending per-repo program above, but you can specify
 whichever events you wish.
 
+Once you have set up your webhook, GitHub will send a ping message, which snare
```

Comment:
How about something a bit simpler like:

> Once you have set up your webhook, GitHub will automatically send a test "ping" message. `snare` acknowledges pings (i.e. confirms to GitHub that it is running), but does not execute any commands in response to pings.

?

Reply:
Yeah, that's more concise.